File: ABC/Python3/ABC126/B.py
S = input()

left = int(S[:2])
right = int(S[2:])
if left == 0:
  if 0 < right <= 12:
    print("YYMM")
  else:
    print("NA")
else:
  if left > 12:
    if 0 < right < 13:
      print("YYMM")
    else:
      print("NA")
  elif left > 0:
    if 0 < right < 13:
      print("AMBIGUOUS")
    else:
      print("MMYY")

File: ABC/ABC126/B.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int S, left, right;
	cin >> S;
	left = S / 100;
	right = S % 100;

	string ans = "NA";
	if ((left > 0 && left < 13) && (right > 0 && right < 13)){
		ans = "AMBIGUOUS";
	}
	else if (right >= 0 && (left > 0 && left < 13)) {
		ans = "MMYY";
	}
	else if (left >= 0 && (right > 0 && right < 13)) {
		ans = "YYMM";
	}

	cout << ans << endl;
}
